diff --git a/Makefile b/Makefile
index 8fea662..6d37e47 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,6 @@
-LDLIBS=-lncursesw
-PREFIX=/usr/local
+LDLIBS=-lncursesw -ltinfo
+CFLAGS+=-I /usr/include/ncursesw
+PREFIX=$(HOME)
 MANPREFIX=$(PREFIX)/man
 BINDIR=$(DESTDIR)$(PREFIX)/bin
 MANDIR=$(DESTDIR)$(MANPREFIX)/man1
diff --git a/config.h b/config.h
index 3665bde..1837cf5 100644
--- a/config.h
+++ b/config.h
@@ -14,50 +14,50 @@
 #define RVK_CD_UP       "h"
 #define RVK_HOME        "H"
 #define RVK_TARGET      "t"
-#define RVK_COPY_PATH   "y"
-#define RVK_PASTE_PATH  "p"
+#define RVK_COPY_PATH   "w"
+#define RVK_PASTE_PATH  "u"
 #define RVK_REFRESH     "r"
-#define RVK_SHELL       "^M"
+#define RVK_SHELL       "s"
 #define RVK_VIEW        " "
 #define RVK_EDIT        "e"
-#define RVK_OPEN        "o"
+#define RVK_OPEN        "^M"
 #define RVK_SEARCH      "/"
-#define RVK_TG_FILES    "f"
-#define RVK_TG_DIRS     "d"
-#define RVK_TG_HIDDEN   "s"
-#define RVK_NEW_FILE    "n"
-#define RVK_NEW_DIR     "N"
-#define RVK_RENAME      "R"
+#define RVK_TG_FILES    "F"
+#define RVK_TG_DIRS     "D"
+#define RVK_TG_HIDDEN   "."
+#define RVK_NEW_FILE    "N"
+#define RVK_NEW_DIR     "n"
+#define RVK_RENAME      "c"
 #define RVK_TG_EXEC     "E"
-#define RVK_DELETE      "D"
-#define RVK_TG_MARK     "m"
-#define RVK_INVMARK     "M"
+#define RVK_DELETE      "d"
+#define RVK_TG_MARK     "y"
+#define RVK_INVMARK     "Y"
 #define RVK_MARKALL     "a"
-#define RVK_MARK_DELETE "X"
-#define RVK_MARK_COPY   "C"
-#define RVK_MARK_MOVE   "V"
+#define RVK_MARK_DELETE "x"
+#define RVK_MARK_COPY   "p"
+#define RVK_MARK_MOVE   "m"
 
 /* Colors available: DEFAULT, RED, GREEN, YELLOW, BLUE, CYAN, MAGENTA, WHITE, BLACK. */
-#define RVC_CWD         GREEN
+#define RVC_CWD         YELLOW
 #define RVC_STATUS      CYAN
 #define RVC_BORDER      BLUE
-#define RVC_SCROLLBAR   CYAN
+#define RVC_SCROLLBAR   MAGENTA
 #define RVC_LINK        CYAN
 #define RVC_HIDDEN      YELLOW
 #define RVC_EXEC        GREEN
 #define RVC_REG         DEFAULT
-#define RVC_DIR         DEFAULT
+#define RVC_DIR         BLUE
 #define RVC_CHR         MAGENTA
 #define RVC_BLK         MAGENTA
 #define RVC_FIFO        BLUE
 #define RVC_SOCK        MAGENTA
-#define RVC_PROMPT      DEFAULT
-#define RVC_TABNUM      DEFAULT
+#define RVC_PROMPT      GREEN
+#define RVC_TABNUM      MAGENTA
 #define RVC_MARKS       YELLOW
 
 /* Special symbols used by the TUI. See <curses.h> for available constants. */
-#define RVS_SCROLLBAR   ACS_CKBOARD
-#define RVS_MARK        ACS_DIAMOND
+#define RVS_SCROLLBAR   ACS_VLINE
+#define RVS_MARK        ACS_BULLET
 
 /* Prompt strings for line input. */
 #define RV_PROMPT(S)    S ": "
@@ -85,4 +85,4 @@
    process each time an external  program is invoked. Leave this macro
    undefined if you prefer external  programs to be launched with just
    `$EXTERNAL_PROGRAM [arg]`. */
-#define RV_SHELL        "/bin/sh"
+/*#define RV_SHELL        "/bin/sh"*/
diff --git a/rover.c b/rover.c
index 723c11a..b566422 100644
--- a/rover.c
+++ b/rover.c
@@ -1411,7 +1411,7 @@ paste_path_fail:
         } else if (!strcmp(key, RVK_DELETE)) {
             if (rover.nfiles) {
                 message(YELLOW, "Delete \"%s\"? (Y/n)", ENAME(ESEL));
-                if (rover_getch() == 'Y') {
+                if (rover_getch() == 'y') {
                     const char *name = ENAME(ESEL);
                     int ret = ISDIR(ENAME(ESEL)) ? deldir(name) : delfile(name);
                     reload();
@@ -1448,7 +1448,7 @@ paste_path_fail:
         } else if (!strcmp(key, RVK_MARK_DELETE)) {
             if (rover.marks.nentries) {
                 message(YELLOW, "Delete all marked entries? (Y/n)");
-                if (rover_getch() == 'Y')
+                if (rover_getch() == 'y')
                     process_marked(NULL, delfile, deldir, "Deleting", "Deleted");
                 else
                     clear_message();
